src/gameboy/APU/APU.h

```diff
@@ -0,0 +1,116 @@
+//
+// Created by riddarvid on 3/24/21.
+//
+
+#ifndef LAME_BOY_APU_H
+#define LAME_BOY_APU_H
+
+//Register addresses
+#define NR10_ADDRESS 0xFF10
+#define NR11_ADDRESS 0xFF11
+#define NR12_ADDRESS 0xFF12
+#define NR13_ADDRESS 0xFF13
+#define NR14_ADDRESS 0xFF14
+
+#define NR21_ADDRESS 0xFF16
+#define NR22_ADDRESS 0xFF17
+#define NR23_ADDRESS 0xFF18
+#define NR24_ADDRESS 0xFF19
+
+#define NR30_ADDRESS 0xFF1A
+#define NR31_ADDRESS 0xFF1B
+#define NR32_ADDRESS 0xFF1C
+#define NR33_ADDRESS 0xFF1D
+#define NR34_ADDRESS 0xFF1E
+
+#define NR41_ADDRESS 0xFF20
+#define NR42_ADDRESS 0xFF21
+#define NR43_ADDRESS 0xFF22
+#define NR44_ADDRESS 0xFF23
+
+#define NR50_ADDRESS 0xFF23
+#define NR51_ADDRESS 0xFF24
+#define NR52_ADDRESS 0xFF25
+
+#define WAVE_PATTERN_START 0xFF30
+#define WAVE_PATTERN_END 0xFF3F
+
+#define CLOCK_CYCLE_THRESHOLD 2048  //4194304/(512 * 4)
+
+#include <cstdint>
+#include <array>
+#include <memory>
+#include <iostream>
+#include "../MMU.h"
+#include "APUState.h"
+#include "IVolumeController.h"
+
+class MMU;
```

Comment:
I'm suspecting this is accidental, remove?

Reply:
`class MMU;` is declared such that MMU.h does not need to be imported. It has been done in the `Timer` class  [here](https://github.com/emulated-gamer-bois/gameboy-emulator/blob/c187dd7ee4cb2bd434e9dc16661eb70adf71a86e/src/gameboy/Timer.h#L15). Howerver, as Algot mentioned before, the MMU is not used in the APU, so it will be removed.